.gitlab-ci.yml

```
@@ -0,0 +1,27 @@
+image: gitlab-registry.cern.ch/alu/dirac/docker-compose
+
+services:
+  - name: docker:18-dind
+
+before_script:
+  - "echo ${CI_REGISTRY_PASSWORD} | docker login -u ${CI_REGISTRY_USER} ${CI_REGISTRY} --password-stdin"
+
+variables:
+  COMPOSE_HTTP_TIMEOUT: "360"
+
+test:
```

This allows us to use the shared runners rather than maintaining our own:
```suggestion
test:
  tags:
    - docker-privileged
```